file-updates/test.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env sh
+
+trap teardown EXIT
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+DEV_KUBECONFIG="--kubeconfig=$SCRIPTDIR/stage_dev_release_kubelogin"
+MANAGED_KUBECONFIG="--kubeconfig=$SCRIPTDIR/stage_managed_release_kubelogin"
+
+MANAGED_NAMESPACE="managed-release-team-tenant"
+APPLICATION_NAME="fileupdatestest"
+COMPONENT_NAME="fileupdatestest-component"
+RELEASE_PLAN_NAME="file-updates-test-rp"
+RELEASE_PLAN_ADMISSION_NAME="file-updates-test-rpa"
+RELEASE_STRATEGY_NAME="file-updates-test-push-rs"
```

Comment:
I am confused as to why this commit adds two release strategies (and creates them), but the releaseplanadmission obviously just uses one

Reply:
removed un-referenced release strategy